# HG changeset patch
# User Mike Miller <mtmiller@octave.org>
# Date 1465764402 25200
#      Sun Jun 12 13:46:42 2016 -0700
# Node ID 1315aa0ca46658e81a2679ad4a78f98f5ecd2d62
# Parent  67e51c889f34f2f96473558ef94b7cce9dfb1a45
build: Ensure gnuplot_binary.m is generated before building docs (bug #48202)

* scripts/module.mk (OCTAVE_INTERPRETER_TARGETS): Include $(GEN_FCN_FILES)
in the list.

diff --git a/scripts/module.mk b/scripts/module.mk
--- a/scripts/module.mk
+++ b/scripts/module.mk
@@ -79,16 +79,19 @@ image_DATA += $(SCRIPTS_IMAGES)
 
 GEN_FCN_FILES_IN = $(GEN_FCN_FILES:.m=.in)
 
 ALL_LOCAL_TARGETS += \
   $(GEN_FCN_FILES) \
   $(PKG_ADD_FILES) \
   $(JAR_FILES)
 
+OCTAVE_INTERPRETER_TARGETS += \
+  $(GEN_FCN_FILES)
+
 define PKG_ADD_FILE_TEMPLATE
 $(1)/PKG_ADD: $$($(2)_FCN_FILES) $$($(2)_GEN_FCN_FILES) $(1)/$(octave_dirstamp) scripts/mk-pkg-add
 	$$(AM_V_GEN)rm -f $$@-t $$@ && \
 	$$(SHELL) $$(srcdir)/scripts/mk-pkg-add $(srcdir) $$($(2)_FCN_FILES) -- $$($(2)_GEN_FCN_FILES) > $$@-t && \
 	mv $$@-t $$@
 endef
 
 $(foreach d, $(FCN_FILE_DIRS), $(eval $(call PKG_ADD_FILE_TEMPLATE, $(d),$(subst /,_,$(subst -,_,$(d))))))
